src/System.Private.ServiceModel/tests/Scenarios/SelfHostWcfService/ConfigBackup.xml

```diff
@@ -0,0 +1,176 @@
+﻿<configuration>
+  <!-- 
+    This is a copy of the old app.config for reference.
+    My intention is to replace all the XML config with code.
```

Comment:
Both Matt and Stephen expressed that they preferred creating the bindings in code rather than config. My intention is that there would be no recognition of a config file in the bridge. The backup xml file here is just my reference as I'm converting this over.